snuba/cli/config.py

```diff
@@ -1,7 +1,8 @@
-import click
 import json
 from typing import Any, Mapping
 
+import click
```

Comment:
should we put a disclaimer at the top of the file that it is being deprecated in favor of the snuba admin portal?